tailwind.config.js

```diff
@@ -38,6 +41,7 @@ module.exports = {
       'blue-focus-sm': `0px 0px 0px 1px ${bluePrimary}, 0px 0px 0px 4px ${blue80}`,
       'red-focus-md': `0px 0px 0px 1px #ffffff, 0px 0px 0px 4px ${red80}`,
       'red-focus-sm': `0px 0px 0px 1px ${redPrimary}, 0px 0px 0px 4px ${red80}`,
+      'neutral-focus-sm': `0px 0px 0px 1px ${neutral20}, 0px 0px 0px 4px ${neutral80}`,
```

Comment:
This is now a smell - these aren't small and medium anymore, they are white and black border.e